tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -199,7 +201,7 @@ dnl ssp1-maxmspk, ssp0-RTHeadset
 
 #SSP 0 (ID: 0)
 DAI_CONFIG(SSP, 0, 0, SSP0-Codec,
-        SSP_CONFIG(I2S, SSP_CLOCK(mclk, 19200000, codec_mclk_in),
+        SSP_CONFIG(I2S, SSP_CLOCK(mclk, SSP_MCLK, codec_mclk_in),
```

Comment:
indeed there are physical restrictions on what can be pushed on the outputs, I don't think anything above 20MHz would work on the MCLK pins.
Can we please check that we never output 38.4 on the MCLK pins in other cases?
Thanks!

Reply:
Thanks for sharing @plbossart 
I am not as surely as you are about that 38.4MHz MCLK doesn't work, we have several places using 38.4, they sounds fine there but maybe the mclk pin could be actually not connected to the codec on those platforms.